cypress.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = defineConfig({
             username: userName.toLowerCase(),
             email: 'test'+`${randomNumber}`+'@mail.com',
             password: '12345Qwert!',
+            bio: faker.lorem.words(),
```

Comment:
Consider using faker to generate a more realistic email instead of concatenating strings.